build/scripts/update_dependencies.csx

```diff
@@ -1,54 +1,28 @@
 #r "System.Net.Http.dll"
+#r "System.Xml.Linq"
 
 using System.Net.Http;
 using System.Runtime.CompilerServices;
 using System.Text.RegularExpressions;
+using System.Xml.Linq;
 
-var knownRepos = new[] { "coreclr", "corefx", "projectk-tfs", "symreader" };
+bool lkg = Args.Remove("/lkg");
+bool help = Args.Remove("/help") || Args.Remove("/?");
 
-if (Args.Count < 1)
+if (help || Args.Count > 0)
 {
-    Console.Error.WriteLine("Usage: fxupdate <Roslyn branch> [/c:<repo>=<commit-sha1>]");
-    Console.Error.WriteLine($"Where <repo> is name of a repo: {string.Join(", ", knownRepos)}");
-    return 1;
-}
-
-string roslynBranch = Args[0];
-
-var commits = 
-    Args.Skip(1).Where(a => a.StartsWith("/c:")).Select(a => a.Substring("/c:".Length)).
-    ToDictionary(k => k.Split('=')[0].Trim(), k => k.Split('=')[1].Trim(), StringComparer.OrdinalIgnoreCase);
-
-string unknownRepo = commits.Keys.FirstOrDefault(k => !knownRepos.Contains(k));
-if (unknownRepo != null)
-{
-    Console.Error.WriteLine($"Unknown repo: {unknownRepo}");
+    Console.Error.WriteLine("Usage: fxupdate [/help] [/lkg]");
+    Console.Error.WriteLine();
+    Console.Error.WriteLine($"/lkg ... if specified the script uses the package versions specified in LKG_Packages.txt files in dotnet/versions repo");
```

Comment:
Perhaps `"/lkg\tuse versions in LKG_Packages.txt files in dotnet/versions (default: Latest_Packages.txt)"`